examples/kyb-app/src/domains/collection-flow/collection-flow.api.ts

```diff
@@ -36,6 +39,18 @@ export const fetchCollectionFlowSchema = async (): Promise<{
   };
 };
 
+export const fetchUISchema = async (): Promise<UISchema> => {
+  const result = await request
+    .get('collection-flow/configuration', {
+      searchParams: {
+        uiContext: 'collection_flow',
+      },
+    })
+    .json<UISchema>();
+
+  return result;
+};
```

Comment:
Prefer a runtime check over passing a type argument to `json` and setting a return type. These types would need to be maintained and if the runtime changes it would be easier to miss.